message_ix_models/model/material/data_methanol_new.py

```diff
@@ -184,7 +184,7 @@ def broadcast_reduced_df(df, par_name):
 
         for col in yr_col_inp:
             yr_cols_codes[col] = literal_eval(df_bc_node[col].values[0])
-            broadcast_years(df_bc_node, yr_col_out, yr_cols_codes, col)
+            df_bc_node = broadcast_years(df_bc_node, yr_col_out, yr_cols_codes, col)
```

Comment:
The function called here has no tests or docstring. Does it do something that can't be done with [`message_ix_models.util.broadcast()`](https://docs.messageix.org/projects/models/en/latest/api/util.html#message_ix_models.util.broadcast)? If so, what?

Reply:
I renamed that top level function now to a imo better name and added docstrings to all functions in this module. `broadcast_years()` and `broadcast_nodes()` use `message_ix_models.util.broadcast()` to unpivot the rows that are stored in `methanol_techno_economic.xlsx`.